pkgs/development/interpreters/python/pypy/default.nix

```diff
@@ -122,7 +123,7 @@ in with passthru; stdenv.mkDerivation rec {
     ln -s $out/${executable}-c/lib-python/${if isPy3k then "3" else pythonVersion} $out/lib/${libPrefix}
 
     # Include a sitecustomize.py file
-    cp ${../sitecustomize.py} $out/lib/${libPrefix}/${sitePackages}/sitecustomize.py
+    cp ${../sitecustomize.py} $out/${if isPy38OrNewer then sitePackages else "lib/${libPrefix}/${sitePackages}"}/sitecustomize.py
```

Comment:
should this if .... maybe be moved into the `passthru.sitePackages`?

Reply:
I don't think that's possible in this context/how `sitePackages` is used throughout the tree?